src/commands/flags.ts

```diff
@@ -1617,6 +1628,7 @@ export class Flags {
     Flags.applicationEnv,
     Flags.applicationProperties,
     Flags.bootstrapProperties,
+    Flags.genesisThrottlesFile,
```

Comment:
I made these alphabetical since the list is so long.

Reply:
fixed